src/reducers/versions.tsx

```diff
@@ -423,6 +423,68 @@ export const isFileLoading = (
   return false;
 };
 
+export const getDiffAnchors = (diff: DiffInfo): string[] => {
+  const anchors: string[] = [];
+
+  for (const hunk of diff.hunks) {
+    let seekingChange = true;
+    for (const change of hunk.changes) {
+      if (change.type === 'normal') {
+        seekingChange = true;
+      } else if (seekingChange) {
```

Comment:
I'm confused as to why this ignores all the changes that come before the first `normal` change. Why not just filter out all normal changes? I know there is a reason because I tried it and the tests failed, heh. Could you explain? I think if you were to rewrite the tests with my suggestion for a different approach then it would become clear.

Reply:
This does not ignore all changes that come before the first `normal` change, as the test bears out. I'm not sure why you think it does. Or maybe I'm misinterpreting what you're saying. Please take a look at the updated test and see if you still feel this problem exists.

As for why I cannot just filter out all `normal` changes, if I did that then I'd have no way to identify *blocks* of changes, which is really the point of this function.